SummerProject/static/src/components/Footer/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './style.css';
+
+function Footer() {
+        return (
+            <div className="footer">
+                <footer className="section footer-classic context-dark bg-image">
+                    <div className="container">
+                        <div className="row row-30">
+                            <div className="col-md-4 col-xl-3">
+                                <div>
+                                    <h5 className="title">COMPANY INFO</h5>
+                                </div>
+                                <ul className="nav-list">
+                                    <li><a href="#">About Sport News</a></li>
+                                    <li><a href="#">News/In the Press</a></li>
+                                    <li><a href="#">Advertising</a></li>
+                                    <li><a href="#">Events</a></li>
+                                    <li><a href="#">Contact Us</a></li>
+                                </ul>
+                            </div>
+                            <div className="col-md-4">
+                                <div>
+                                    <h5 className="title">CONTRIBUTORS</h5>
+                                </div>
+                                <ul className="nav-list">
+                                    <li><a href="#">Featured Writers Program</a></li>
+                                    <li><a href="#">Featured Team Writers Program</a></li>
+                                    <li><a href="#">Internship Program</a></li>
+                                </ul>
+                            </div>
+                            <div className="col-md-4 col-xl-3">
+                                <div>
+                                    <h5 className="title">NEWSLETTER</h5>
+                                </div>
+                                <ul className="nav-list">
+                                    <li><a href="#">Sign up to recieve the latest in sport news</a></li>
+                                    <form className="subscribe-form">
+                                        <div className="subscribe">
+                                            <input type="text" placeholder="Your email address"/>
+                                            <input type="submit" value="Subscribe"/>
```

Comment:
No email validation, u must add it (u can see how button can change in Figma)

Reply:
I will add it when u merged ur commit

Reply:
We'll create a separate story for implementing Subscribe functionality